cfgov/regulations3k/models/django.py

```diff
@@ -241,6 +241,13 @@ class Section(models.Model):
         FieldPanel('contents'),
     ]
 
+    @property
```

Comment:
I think this is fine as an overall approach, with one caveat.

I waffled a bit on whether this should be something like a template tag or a property on the model. I think a property on the model makes more sense than a template tag if it's a [`cached_property`](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.functional.cached_property).